g2p/mappings/utils.py

```diff
@@ -389,6 +389,10 @@ def load_from_file(path: Union[Path, str]) -> list:
         raise exceptions.IncorrectFileType(
             f"File {path} is not a valid mapping filetype."
         )
+    if not mapping:
+        raise exceptions.MalformedMapping(
```

Comment:
also need a unit test to verify that this gets raised properly

Reply:
Good point. This is the very bug that triggered this line of work, but I didn't make a unit test for it. Thanks for pointing it out!